.github/workflows/pytest_coverage.yml

```
@@ -0,0 +1,60 @@
+        uses: actions/setup-python@v5.6.0
+        with:
+          python-version: 3.x
+
+      - name: Cache pip dependencies
+        uses: actions/cache@v4.2.3
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)